src/calendar.cpp

```diff
@@ -282,39 +282,81 @@ float calendar::sunlight() const
     }
 }
 
-std::string calendar::print_duration( int turns )
+std::string calendar::print_clipped_duration( int turns )
 {
-    std::string res;
+    if( turns >= INDEFINITELY_LONG ) {
+        return _( "forewer" );
```

Comment:
Is this intended? Would be better to write a line of comment something like `//~ intended spelling` to remind the translators, or our translators might think of it as a typo and report it as an issue on Transifex.